esignet-default.properties

```diff
@@ -243,6 +243,7 @@ mosip.esignet.cache.expire-in-seconds={'clientdetails' : 86400, \
 
 mosip.esignet.domain.url=https://${mosip.esignet.host}
 mosip.esignet.discovery.issuer-id=${mosip.esignet.domain.url}${server.servlet.path}
+mosip.esignet.file.url=https://${mosip.api.host}
```

Comment:
can you rename this to mosip.esignet.api.url ? Because this has not nothing specific to file.

Reply:
done